pkg/events/sorting/cpu_queue.go

```diff
@@ -34,9 +34,9 @@ func (cq *cpuEventsQueue) InsertByTimestamp(newEvent *trace.Event) error {
 				break
 			}
 			if insertLocation.next == insertLocation {
-				err := fmt.Errorf("encountered node with self reference at next")
-				if allocErr != nil {
-					err = fmt.Errorf("%s\n%s", allocErr, err)
+				if err != nil {
+					err2 := "encountered node with self reference at next"
```

Comment:
The err2 holds string literal, so why it's not part of the pattern?
```suggestion
					err = fmt.Errorf("encountered node with self reference at next: %w", err)
```

Reply:
I kept it like this to reduce line width. Golang likes to be very verbose in columns-wise =D. I'll change it, its ok.